### PR TITLE
Add support for document store

### PR DESCRIPTION
**Pull request by mayeonnaise**
Closes #93 
Adds support for document store

### CODE REVIEW

File: src/components/databaseUi/DocStoreUI.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    </div>);
+  }
+
+  getDataDisplay(index: any) {
+    return Object.keys(index._index).map((key) => {
+      console.log(index._index[key]);
+      return { Document: JSON.stringify(index._index[key].payload.value) }
+    })
+  }
+}
```
Comment (hannessolo) [inline src/components/databaseUi/DocStoreUI.tsx]: No new line but no big deal

File: src/components/viewDatabase/Sidebar.module.css [hunk 2]
```diff
@@ -146,6 +146,48 @@
...
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  outline: none;
+  text-align:center;
+}
+
 .inputFieldSubmit:hover {
   background-color: #0A3464FF;
 }
```
Comment (hannessolo) [inline src/components/viewDatabase/Sidebar.module.css]: same here

File: src/components/databaseUi/DocStoreUI.tsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+          <div style={{display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', marginTop: '0.5vh'}}>
+            <input type="submit" value={opName} className={ToolbarStyle.inputFieldSubmit}/>
+          </div>
+        </form>
+      </div>);
+    };
+
+    const addEditorField = (opName, op) => {
+      return () => (<div className={ToolbarStyle.inputFieldContainer}>
+        <form onSubmit= {op}>
```
Comment (justincqz) [inline src/components/databaseUi/DocStoreUI.tsx]: e.preventDefault() might be needed cause form submits traditionally cause page refresh, which is not what we want

File: src/components/viewDatabase/DAGNodeTooltip.tsx [hunk 4]
```diff
@@ -30,11 +30,14 @@ const DAGNodeTooltip: React.FC<{nodeInfo: any, rect: ClientRect}> = ({nodeInfo,
       {
         Object.keys(nodeInfo.payload)
         .filter((key) => nodeInfo.payload[key] != null)
-        .map((key) => (
-          <div key={key}>
-            {key}: {nodeInfo.payload[key]}
-          </div>
-        ))
+        .map((key) => {
+          console.log(nodeInfo);
```
Comment (justincqz) [inline src/components/viewDatabase/DAGNodeTooltip.tsx]: Remove the debug console log?

File: src/components/databaseUi/DocStoreUI.tsx (hunk 3 above)
Comment (mayeonnaise) [reply src/components/databaseUi/DocStoreUI.tsx]: it has, i give it submit document which has an e.preventdefault

**Pull request merged**